README.md

```diff
@@ -88,3 +88,45 @@ The following flags are available for `organizations find`:
 * Find an organization by its externalID: `ocm support organizations find [organizationExternalID]`
 * Find an organization and include its subscriptions: `ocm support organizations find [organizationID] --fetchSubscriptions`
 * Find an organization and include its quota: `ocm support organizations find [organizationID] --fetchQuota`
+
+### RegistryCredentials
+
+The `registryCredentials` command gets information about or execute actions on registry credentials.
+
+#### Creating registry credentials
+
+Use the `create` subcommand to create registry credentials for current account. 
+
+The following flags are available for `registryCredentials create`:
+
+```
+-h, --help                   help for find
```

Comment:
should it be "help for create" instead? Since we don't have any flags for create, I think we can safely remove the flags section here.